src/main.py

```diff
@@ -135,6 +135,9 @@ async def handle_event(event: Dict, event_context, projects_ids: Optional[List[s
         disabled_apis = {}
         for project_id in projects_ids:
             disabled_apis = {project_id: await get_all_disabled_apis(context, token, project_id)}
+            if disabled_apis[project_id] == {project_id}:
+                projects_ids.remove(project_id)
+                context.log(f"monitoring.googleapis.com API disabled in the project: {project_id}, this project will not be monitored")
```

Comment:
(minor) hope the customer won't have hundreds of such projects (with monitoring api disabled) ;) we could collect these project ids first and then log them once.